src/code42cli/cmds/search/extraction.py

```diff
@@ -72,9 +75,7 @@ def create_handlers(
     extractor = extractor_class(sdk, ExtractionHandlers())
     handlers = _set_handlers(cursor_store, checkpoint_name)
 
-    @warn_interrupt(
-        warning="Attempting to cancel cleanly to keep checkpoint data accurate. One moment..."
-    )
+    @warn_interrupt(warning=INTERRUPT_WARNING)
```

Comment:
An observation. If the interrupt warning would be printed when "CTRL-C" is pressed after command execution then it isn't, atleast for me.

Reply:
Like if you hit CTRL-C twice? I think that is expected

Reply:
found this in while executing `search` command with  `saved-search` option. It works.

Reply:
I never see the interrupt warning, weird... probably address this outside this PR